project/plugins.sbt

```diff
@@ -4,6 +4,6 @@ addSbtPlugin("com.github.alonsodomin" % "sbt-spark" % "0.6.0")
 
 addSbtPlugin("ch.jodersky" % "sbt-jni" % "1.4.1")
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
```

Comment:
0.15.0 introduced some incompatibilities that were causing `build/sbt assembly` to fail.

Reply:
What were the incompatibilities?

Reply:
Was getting `java.lang.NoSuchMethodError: sbtassembly.AssemblyOption.copy$default$14()Lscala/Enumeration$Value;` in the sbtspark package.